generate_adapter.py

```diff
@@ -95,16 +93,15 @@ def main(
         top_k=top_k,
         eos_id=tokenizer.eos_id
     )
+    t = time.perf_counter() - t0
 
     output = tokenizer.decode(output)
     output = output.split("### Response:")[1].strip()
-
     print(output)
```

Comment:
@awaelchli did you intend to include the decoding + splitting + print inside the timer?

Reply:
Yes, I think it should be, because that's the end result the user is waiting for in the end. print and str.split will be negligible.